src/otx/algo/detection/ssd.py

```diff
@@ -572,21 +601,42 @@ def _exporter(self) -> OTXModelExporter:
 
         mean, std = get_mean_std_from_data_processing(self.config)
 
-        with self.export_model_forward_context():
-            return MMdeployExporter(
-                model_builder=self._create_model,
-                model_cfg=deepcopy(self.config),
-                deploy_cfg="otx.algo.detection.mmdeploy.ssd_mobilenetv2",
-                test_pipeline=self._make_fake_test_pipeline(),
-                task_level_export_parameters=self._export_parameters,
-                input_size=self.image_size,
-                mean=mean,
-                std=std,
-                resize_mode="standard",
-                pad_value=0,
-                swap_rgb=False,
-                output_names=["feature_vector", "saliency_map"] if self.explain_mode else None,
-            )
+        return OTXNativeModelExporter(
+            task_level_export_parameters=self._export_parameters,
+            input_size=self.image_size,
+            mean=mean,
+            std=std,
+            resize_mode="standard",
+            pad_value=0,
+            swap_rgb=False,
+            via_onnx=True,  # Currently SSD should be exported through ONNX
+            onnx_export_configuration={
+                "input_names": ["image"],
+                "output_names": ["boxes", "labels"],
+                "dynamic_axes": {
+                    "image": {0: "batch", 2: "height", 3: "width"},
+                    "boxes": {0: "batch", 1: "num_dets"},
+                    "labels": {0: "batch", 1: "num_dets"},
+                },
+                "autograd_inlining": False,
+            },
+            output_names=["feature_vector", "saliency_map"] if self.explain_mode else None,
+        )
+
+    def forward_for_tracing(self, inputs: Tensor) -> list[InstanceData]:
```

Comment:
I didn't see any usage of `forward_for_tracing` in the PR. Also, I couldn't locate it within OTXNativeModelExporter either. Could you provide more details?

Reply:
I don't search for the details of forward_for_tracing, but I believe this is the forward path definition when the models are exported

Reply:
Thank you, Jaeguk. I've noticed a monkey patch beneath OTXModel.export.
```
self.forward = self.forward_for_tracing  # type: ignore[method-assign, assignment]
return self._exporter.export(
    self,
    output_dir,
    base_name,
    export_format,
    precision,
)
```
Since there's no documentation available, I found myself a bit confused regarding the purpose of this method. 
Besides, I'm concerned about the potential risks associated with such monkey patching, especially without any warnings or documentation. While we may not have a better solution, it would be good to mark this with a TODO and prioritize improving the documentation before resorting to monkey-patching export functions, if possible of course.

Reply:
@vinnamkim Do you have any opinion for this?